Cargo.toml

```diff
@@ -41,6 +41,7 @@ tiny-keccak = "1.3"
 tokio-core = "0.1.10"
 tokio-io = "0.1.3"
 unwrap = "1.1"
+url = "1.5.1"
```

Comment:
I believe we have decided to use ~ requirements for versions: http://doc.crates.io/specifying-dependencies.html#tilde-requirements 
hence `url = "~1.5.1` in this case
Also, see this: https://github.com/maidsafe/routing/blob/master/Cargo.toml